components/onboarding_tasklist/onboarding_tasklist_completed.tsx

```diff
@@ -122,26 +142,16 @@ const Completed = (props: Props): JSX.Element => {
                             defaultMessage='Well done. You’ve completed all of the tasks!'
                         />
                     </h2>
-                    <p>
+                    <span className='completed-subtitle'>
                         <FormattedMessage
                             id={'onboardingTask.checklist.completed_subtitle'}
-                            defaultMessage='We hope Mattermost is more familiar now. Need more help? '
+                            defaultMessage='We hope Mattermost is more familiar now.'
                         />
-                        <a
-                            target='_blank'
-                            rel='noopener noreferrer'
-                            href={'https://docs.mattermost.com/'}
-                        >
-                            <FormattedMessage
-                                id='onboardingTask.checklist.documentation_link'
-                                defaultMessage='See our documentation.'
-                            />
-                        </a>
-                    </p>
```

Comment:
Is this help link removal intended?

Reply:
yes, according to the design this gets removed.
<img width="359" alt="Captura de pantalla 2022-03-03 a las 7 50 17" src="https://user-images.githubusercontent.com/10082627/156568047-156bbcfe-7337-48a5-b3bc-e4a3fb512a21.png">